.travis.yml

```
@@ -19,6 +19,7 @@ matrix:
   - ghc: 8.2.2
   - ghc: 8.4.4
   - ghc: 8.6.5
+  - ghc: 8.8.1
 
   # stack
   - ghc: 8.6.5
```

Maybe here also should be `ghc-8.8.1` since stack resolver is updated
```suggestion
   - ghc: 8.8.1
```